# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/about/nsAboutCache.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/protocol/about/nsAboutCache.cpp
--- a/netwerk/protocol/about/nsAboutCache.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/protocol/about/nsAboutCache.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -126,8 +126,7 @@
   return NS_OK;
 }
 
-NS_IMETHODIMP nsAboutCache::Channel::AsyncOpen(nsIStreamListener *aListener,
-                                               nsISupports *aContext) {
+NS_IMETHODIMP nsAboutCache::Channel::AsyncOpen(nsIStreamListener *aListener) {
   nsresult rv;
 
   if (!mChannel) {
@@ -138,25 +137,16 @@
   rv = VisitNextStorage();
   if (NS_FAILED(rv)) return rv;
 
-  MOZ_ASSERT(!aContext, "asyncOpen2() does not take a context argument");
-  rv = NS_MaybeOpenChannelUsingAsyncOpen2(mChannel, aListener);
+  rv = NS_MaybeOpenChannelUsingAsyncOpen(mChannel, aListener);
   if (NS_FAILED(rv)) return rv;
 
   return NS_OK;
 }
 
-NS_IMETHODIMP nsAboutCache::Channel::AsyncOpen2(nsIStreamListener *aListener) {
-  return AsyncOpen(aListener, nullptr);
-}
-
 NS_IMETHODIMP nsAboutCache::Channel::Open(nsIInputStream **_retval) {
   return NS_ERROR_NOT_IMPLEMENTED;
 }
 
-NS_IMETHODIMP nsAboutCache::Channel::Open2(nsIInputStream **_retval) {
-  return NS_ERROR_NOT_IMPLEMENTED;
-}
-
 nsresult nsAboutCache::Channel::ParseURI(nsIURI *uri, nsACString &storage) {
   //
   // about:cache[?storage=<storage-name>[&context=<context-key>]]